editoast/editoast_schemas/Cargo.toml

```diff
@@ -8,8 +8,11 @@ license = "LGPL-3.0"
 derivative.workspace = true
 editoast_common.workspace = true
 enum-map.workspace = true
+geos.workspace = true
```

Comment:
We don't use `geos` features in schemas, only its re-export of the crate `geojson`. I'd suggest you use `geojson` directly in schema to keep the dependency tree as small as possible.

Reply:
Done.